src/sdltiles.cpp

```diff
@@ -3781,6 +3781,14 @@ window_dimensions get_window_dimensions( const point &pos, const point &size )
     return get_window_dimensions( {}, pos, size );
 }
 
+bool window_contains_point_relative( const catacurses::window &win, const point &p )
```

Comment:
It is guarded by TILES defines and thus breaks CURSES builds.

Reply:
what's your suggested solution? is it ok to just move all related methods out of the TILES define?

Reply:
Yes, it should be independent from renderer used - you've used this function both in tiles and non-tiles code, so it should not be guarded.